openapi-cli/src/test/resources/ballerina-to-openapi/response/nil_return_type.bal

```diff
@@ -0,0 +1,80 @@
+import ballerina/http;
+
+service /payloadV on new http:Listener(9090) {
+    resource function get path1() {
+    };
+    resource function get path_with_query(string id)  {
+    };
+    resource function get path_with_path/[string id]() {
+    };
+    resource function get path_with_headers(@http:Header string header) {
+    };
+    // resource function get path_with_request_body(@http:Payload string payload) {
+    // };
+
+    resource function post path1() {
+    };
+    resource function post path_with_query(string id)  {
+    };
+    resource function post path_with_path/[string id]() {
+    };
+    resource function post path_with_headers(@http:Header string header) {
+    };
+    resource function post path_with_request_body(@http:Payload string payload) {
+    };
+
+    resource function put path1() {
+    };
+    resource function put path_with_query(string id)  {
+    };
+    resource function put path_with_path/[string id]() {
+    };
+    resource function put path_with_headers(@http:Header string header) {
+    };
+    resource function put path_with_request_body(@http:Payload string payload) {
+    };
+
+    resource function patch path1() {
+    };
+    resource function patch path_with_query(string id)  {
+    };
+    resource function patch path_with_path/[string id]() {
+    };
+    resource function patch path_with_headers(@http:Header string header) {
+    };
+    resource function patch path_with_request_body(@http:Payload string payload) {
+    };
+
+    resource function delete path1() {
+    };
+    resource function delete path_with_query(string id)  {
+    };
+    resource function delete path_with_path/[string id]() {
+    };
+    resource function delete path_with_headers(@http:Header string header) {
+    };
+    resource function delete path_with_request_body(@http:Payload string payload) {
+    };
+
+    resource function head path1() {
+    };
+    resource function head path_with_query(string id)  {
+    };
+    resource function head path_with_path/[string id]() {
+    };
+    resource function head path_with_headers(@http:Header string header) {
+    };
+    // resource function head path_with_request_body(@http:Payload string payload) {
+    // };
```

Comment:
Any reason to disable this request body scenario?

Reply:
Yeah, these http methods `HEAD` and `GET` aren't allowed to have a request body entity payload, therefore these scenarios capture within negative tests https://github.com/ballerina-platform/openapi-tools/pull/1535/commits/2224d4ac06d6b0f8a6e60c6785bf04d520296a06.

Reply:
Got it. In that case we don't need to keep the commented code right?

Reply:
Yeah, we removed it :)